src/lib/hydrate.ts

```diff
@@ -209,25 +220,20 @@ export const hydrate =
               // need to stop hydrating this subtree? Clear it? Add tests.
               throw new Error('unimplemented');
             }
+          } else if (isIterable(value)) {
```

Comment:
Nodes are ordered before iterables in `NodePart.commit`. Does that matter here?

Reply:
We don't currently support creating `Node`s on the server (no full DOM shim), so there's no way for a user to pass them as values, so currently not supporting.  But if we did they would just be lumped in the final else fallback because again, for all those types, all we're doing is just setting `part.value = value`.  Only `noChange`, `TemplateResult`, and `Iterable` are special for this part of the code.